docs/src/installation.md

```diff
@@ -29,8 +50,6 @@ Pkg.build("RCall")
 
 When `R_HOME` is set to `"*"`, RCall.jl will automatically install R for you using [Conda](https://github.com/JuliaPy/Conda.jl).
 
-Should you experience problems with any of these methods, please [open an issue](https://github.com/JuliaStats/RCall.jl/issues/new).
-
 ## Standard installations
 
 If you want to install R yourself, rather than relying on the automatic Conda installation, you can use one of the following options:
```

Comment:
This might be a good spot to call out another advantage of project-specify R via preferences -- some users might have multiple R versions installed and they can now prefer different R versions for different Julia projects.